.eslintrc.js

```diff
@@ -2,8 +2,12 @@ module.exports = {
   "extends": [
     "react-app",
     "prettier/@typescript-eslint",
-    "plugin:prettier/recommended"
+    "plugin:prettier/recommended",
+    "eslint:recommended",
   ],
+  rules: {
+    "prettier/prettier": ["warn"],
```

Comment:
this was giving me an error on somethign that isnt an error. it was erroring where i use `import type`

```
/home/david/src/across/sdk-v2/src/pool/bridgePool.ts                                                            
 5:14  warning  Parsing error: '=' expected  prettier/prettier  
```